lib/services/devices/deviceService.js

```diff
@@ -21,6 +21,7 @@
  * please contact with::daniel.moranjimenez@telefonica.com
  *
  * Modified by: Federico M. Facca - Martel Innovate
```

Comment:
I think this should be:

Contributors: Daniel Calvo - ATOS Research & Innovation 
Federico M. Facca - Martel Innovate

Reply:
We agreed with @fgalan to use this approach (Modified by: xxx) and Federico has also followed the same style (please see https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/devices/deviceService.js in master branch). Do you think that it is more appropriate to use `Contributors`? In that case, we would need to change some of the headers of the master branch...

Reply:
NTC (I guess)